src/js/app.js

```
@@ -4,7 +4,10 @@
 // THEME
 // ══════════════════════════════════════════════
 (function(){
-  const saved = localStorage.getItem('theme') || 'light';
+  let saved = 'light';
+try {
+  saved = globalThis.localStorage?.getItem('theme') || 'light';
+} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)